client/sqs/v2/publisher.go

```diff
@@ -83,7 +84,7 @@ func (c sqsHeadersCarrier) Set(key, val string) {
 }
 
 // injectHeaders injects the SQS headers carrier's headers into the message's attributes.
```

Comment:
Shall we update GoDoc to mention that it will inject the correlation ID only if not already present?

Reply:
I've modified the godoc. Could you take another look? I'm unable to re-request a review from you on GitHub for some reason. :/